types/redux-seamless-immutable/tslint.json

```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
New package cannot override any rules. Use only extends rule and remove everything else and fix errors if any.